src/components/commons/Buttons/cv_buttons.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import './cv_buttons.css';
+import { CV_BUTTON_TYPES } from 'src/constants/cv-buttons-box';
+import CV_Buttons from '.';
+
+interface CvButtonProps {
+  text?: string;
```

Comment:
i think the button should have a label. so making text optional props might not make sense. rather than text, it should be  label.